index.yaml

```diff
@@ -96,9 +96,9 @@ entries:
   bluescape-monitoring-dashboards:
   - apiVersion: v2
     appVersion: 1.16.2
-    created: "2021-04-16T13:16:26.761570181Z"
+    created: "2021-04-23T01:54:01.773036213Z"
     description: A Helm chart for Kubernetes
-    digest: 2ffa451a176f00700f1ffaf9d6671f233ea1171a681e760bebf9b078f9f53923
+    digest: baf1a55b74b540d2775a30d6364b4e6cf381eb68021b45f984512cc90d7da3d3
```

Comment:
@notjames did you mean not to bump the version? if that one was not rolled out yet, that can be fine, but never hurts to bump it.

Reply:
oh crap! Good call out. I'll update it and commit.